README.md

```diff
@@ -45,10 +45,10 @@ Otherwise, install the following software from your OS package manager or the
 individual project sites when necessary.
 
 + Python 2.6 or 2.7 (use 32 bit if you're on Windows see `Alternate steps for Windows` section below)
-+ pip
-+ CouchDB >= 1.0 (1.2 recommended) ([installation instructions][couchdb])
++ pip  (If you use virtualenv (see below) this will be installed automatically)
++ CouchDB >= 1.0 (1.2 recommended) (install from OS package manager (`sudo apt-get install couchdb`) or [here][couchdb])
   - Mac users: note that when installing erlang, you do NOT need to check out an older version of erlang.rb
-+ PostgreSQL >= 8.4 - (install from OS package manager or [here][postgres])
++ PostgreSQL >= 8.4 - (install from OS package manager (`sudo apt-get install postgresql`) or [here][postgres])
```

Comment:
we may want to start saying `9.1`. we currently run `9.1` and are about to upgrade to `9.4`